x-pack/plugins/maps/public/trigger_actions/visualize_geo_field_action.ts

```diff
@@ -43,6 +46,15 @@ export const visualizeGeoFieldAction = createAction<VisualizeFieldContext>({
   execute: async (context) => {
     const { app, path, state } = await getMapsLink(context);
 
+    const usageCollection = getUsageCollection();
+    usageCollection?.reportUiCounter(
+      APP_ID,
+      'visualize_geo_field',
+      context.originatingApp
```

Comment:
Please change to the below so `visualize_geo_field` is not provided in the 3rd argument.
```
context.originatingApp
       ? context.originatingApp
        : ''
```